CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Breaking changes:
 
 Features:
 
+- [#1248](https://github.com/rails-api/active_model_serializers/pull/1248) Add support for JSON API deserialization (@beauby)
```

Comment:
probably would be good to create an issue output of this referencing unresolved issues, comments, what's next, etc.